app/javascript/mastodon/components/icon_button.js

```diff
@@ -20,6 +20,7 @@ export default class IconButton extends React.PureComponent {
     style: PropTypes.object,
     activeStyle: PropTypes.object,
     disabled: PropTypes.bool,
+    privat: PropTypes.bool,
```

Comment:
You probably meant `private`. And you really shouldn't really pass this to shared IconButton component, there's a class name for that I believe.

Reply:
No, `private` is a reserved word in JS.

It's similar to passing `disabled` as a class name. I believe this make the code structure more clear.